src/components/Modal/Modal.tsx

```diff
@@ -115,7 +115,7 @@ export const BaseModal: ChakraComponent<
     (isDefaultType(type) && (onCancel || onConfirm))
   ) {
     console.warn(
-      "NYPL Reservoir Modal: The `onClose`, `onConfirm`, and `onCancel` props have all been set, but they can not all be used together. Only `onClose` will be used, while `onConfirm` and `onCancel` will be ignored."
+      "NYPL Reservoir Modal: A combination of `onClose`, `onConfirm`, `onCancel`, and `confirmButtonLabel` props have all been set, but they can not all be used together. Either pass the `onClose` (with the default type) or `onCancel`, `onConfirm`, and `confirmButtonLabel` (with the confirmation type)."
```

Comment:
I think the warning message is still not quite right.  The condition above is ultimately an `or` statement, so to say that the `props have all been set` is not necessarily true.  I would remove any implication of that sentiment.  Maybe like this:
```
A combination of `onClose`, `onConfirm`, `onCancel`, and `confirmButtonLabel` props have been passed, but they can not be used as they are currently configured. Either pass the `onClose` prop (with the default type) or pass the `onCancel`, `onConfirm`, and `confirmButtonLabel` props (with the confirmation type).
```